### PR TITLE
fix: more stores

### PR DESCRIPTION
**Pull request by teolemon**
### What
- fix: more stores

### CODE REVIEW

File: data/ocr/store_regex.txt [hunk 1]
```diff
@@ -42,6 +51,7 @@ Carrefour
 Carrefour||grand jury
 Casino
 Colruyt
+Conad
```
Comment (raphael0202) [inline data/ocr/store_regex.txt:54]: Conad is already there (hence the failing test)

File: data/ocr/store_regex.txt [hunk 2]
```diff
@@ -10,6 +10,15 @@ Aldi||gold[aä]hren
 Aldi||gutbio
 Aldi||milsani
 Aldi||sprengel
+Aldi||liveGfree
+Aldi||Simply Nature
+Aldi||Specially Selected
```
Comment (raphael0202) [inline data/ocr/store_regex.txt:15]: Specially selected may be detected on many products

File: data/ocr/store_regex.txt [hunk 3]
```diff
@@ -10,6 +10,15 @@ Aldi||gold[aä]hren
...
+Aldi||Little Journey
```
Comment (raphael0202) [inline data/ocr/store_regex.txt:16]: Same thing for little journey or friendly farms

Comment (teolemon) [reply data/ocr/store_regex.txt:15]: those combinations of words are unlikely to happen ? I'm removing simply nature

File: data/ocr/store_regex.txt [hunk 4]
```diff
@@ -10,6 +10,14 @@ Aldi||gold[aä]hren
 Aldi||gutbio
 Aldi||milsani
 Aldi||sprengel
+Aldi||liveGfree
+Aldi||Simply Nature
```
Comment (teolemon) [inline data/ocr/store_regex.txt:14]:
```suggestion
```

File: data/ocr/store_regex.txt (hunk 3 above)
Comment (raphael0202) [reply data/ocr/store_regex.txt:14]: Well technically they can. The thing is, it is autoapplied here. We should either check that it doesn't occur in the dataset, or have a way to have validation from a human.

File: data/ocr/store_regex.txt [hunk 5]
```diff
@@ -10,6 +10,13 @@ Aldi||gold[aä]hren
 Aldi||gutbio
 Aldi||milsani
 Aldi||sprengel
+Aldi||liveGfree
+Aldi||Little Journey
```
Comment (teolemon) [inline data/ocr/store_regex.txt:14]:
```suggestion
```

File: data/ocr/store_regex.txt [hunk 6]
```diff
@@ -10,6 +10,13 @@ Aldi||gold[aä]hren
...
+Aldi||Winking Owl
+Aldi||Ambiano
+Aldi||Barissimo
+Aldi||Friendly Farms
```
Comment (teolemon) [inline data/ocr/store_regex.txt:18]:
```suggestion
```

**Pull request merged**